app/models/post.rb

```diff
@@ -1,2 +1,12 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+
+  def self.update_counter(parameters)
```

Comment:
- [ ] Kindly add associations between posts and likes, posts and comments. `has_many: comments` `has_many: likes`